packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -128,7 +128,17 @@ class HotRunner extends ResidentRunner {
       printError('Error initializing DevFS: $error');
       return 3;
     }
+    Stopwatch initialUpdateDevFSsTimer;
+    if (benchmarkMode) {
+      initialUpdateDevFSsTimer = new Stopwatch();
+      initialUpdateDevFSsTimer.start();
```

Comment:
nit: it is common to use `..start()` to start a timer on the same line as where it is initialized. In fact, I'd even remove the `if (benchmarkMode)` and always started a `Stopwatch`. It makes no difference performance-wise, and turns 5 line from 131 to 135 into one line:

```dart
final Stopwatch initialiUpdateDevFSsTimer = new Stopwatch()..start();
```